src/Route/AdminPoolLoader.php

```diff
@@ -48,12 +48,24 @@ public function __construct(Pool $pool, array $adminServiceIds, ContainerInterfa
         $this->container = $container;
     }
 
-    public function supports($resource, $type = null)
+    /**
+     * NEXT_MAJOR: Add the ?string param typehint when Symfony 4 support is dropped.
```

Comment:
IMO we should find another comment instead of `NEXT_MAJOR` when it's about dropping support of packages because it could be confusing, moreover, adding them in `master`, related: https://github.com/sonata-project/SonataMediaBundle/pull/1814#discussion_r492047513.

It would be nice to include the standard name of the specific package in the message, like in this case `symfony/config`.

Reply:
@franmomu 
What do you suggest to put there?
a todo?

Reply:
@franmomu We're currently using this comment elsewhere.

Moreover this change will need a next major since its BC break.

We shouldnt block this PR for this, and we could discuss about this on devkit, wdyt ?

Reply:
I'm not blocking this PR (not requesting changes), just a comment because I feel we will have to come back to change these comments.

This won't be BC break since the class is final.